database/upgrade_scripts/020-csaf_tables2.sql

```diff
@@ -0,0 +1,67 @@
+DROP TABLE csaf_cves;
+DROP TABLE csaf_products;
+
```

Comment:
we have env var to trigger re-sync, is this necessary?

Reply:
well, this will save 4 MRs (to enable and disable the flag x 2 environments)

Reply:
my idea is that the env var should be used when there is no schema change so we don't add a migration doing only this

but since there is a schema change that implies need to re-sync, we can do it directly in the migration

Reply:
okay